Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift

```diff
@@ -15,7 +15,7 @@ public final class FormToggleItemView: FormValueItemView<Bool, FormToggleItemSty
     private lazy var stackView: UIStackView = {
         let stackView = UIStackView()
         stackView.axis = .horizontal
-        stackView.alignment = .center
+        stackView.alignment = .top
```

Comment:
You sure this no break anything somewhere else ?

Reply:
It's not a big change in design. Otherwise, we can go with center however it looks a bit odd on larger texts

Reply:
Important question - does it looks fine on CardComponent, AffirmComponent, BoletoComponent ?
Maybe better make this a parameter then ?